src/config/redis.js

```diff
@@ -1,19 +1,44 @@
 import Redis from 'ioredis';
+import logger from './logger.js';
 
 export const redisClient = new Redis({
   host: process.env.DRAGONFLY_HOST,
   port: process.env.DRAGONFLY_PORT,
 });
 
 export async function deleteTokensByUserId(userId, excludeToken) {
-  const tokens = (await redisClient
-    .smembers(`user_tokens:${userId}`))
-    .filter((token) => token !== excludeToken);
-  await redisClient.del(tokens);
-  await redisClient.srem(`user_tokens:${userId}`, tokens);
-};
+  try {
+    const tokens = (await redisClient.smembers(`user_tokens:${userId}`)).filter(
+      (token) => token !== excludeToken
+    );
+    if (tokens.length > 0) {
+      await redisClient.del(tokens);
+      await redisClient.srem(`user_tokens:${userId}`, tokens);
+    }
+  } catch (error) {
+    logger.error('Error deleting tokens for user', {
+      method: 'DELETE',
```

Comment:
This should be the method of the request. If we don't have one, just provide the error mesage